scrapy/utils/iterators.py

```diff
@@ -48,7 +48,7 @@ def xmliter_lxml(obj, nodename, namespace=None, prefix='x'):
     iterable = etree.iterparse(reader, tag=tag, encoding=reader.encoding)
     selxpath = '//' + ('%s:%s' % (prefix, nodename) if namespace else nodename)
     for _, node in iterable:
-        nodetext = etree.tostring(node)
+        nodetext = etree.tostring(node, encoding=six.text_type)
```

Comment:
Right. I guess the "encoding" argument name is misleading. This is to tell [lxml.etree.tostring](http://lxml.de/api/lxml.etree-module.html#tostring) to using Unicode

> Defaults to ASCII encoding without XML declaration. This behaviour can be configured with the keyword arguments 'encoding' (string) and 'xml_declaration' (bool). Note that changing the encoding to a non UTF-8 compatible encoding will enable a declaration by default.
> **You can also serialise to a Unicode string without declaration by passing the unicode function as encoding (or str in Py3), or the name 'unicode'**. This changes the return value from a byte string to an unencoded unicode string.

Without this, I had trouble making the XPaths work. Let me check again.

One can also use `encoding='unicode'` to make is less ambiguous